lib/apiservers/engine/backends/backends.go

```diff
@@ -26,12 +26,12 @@ import (
 
 	log "github.com/Sirupsen/logrus"
 	"github.com/docker/docker/registry"
-	httptransport "github.com/go-swagger/go-swagger/httpkit/client"
-	"github.com/go-swagger/go-swagger/swag"
+	rc "github.com/go-openapi/runtime/client"
+	"github.com/go-openapi/swag"
 
 	"github.com/vmware/vic/lib/apiservers/engine/backends/cache"
 	"github.com/vmware/vic/lib/apiservers/engine/backends/container"
-	"github.com/vmware/vic/lib/apiservers/portlayer/client"
+	apiclient "github.com/vmware/vic/lib/apiservers/portlayer/client"
```

Comment:
`apiclient` isn't really necessary to migrate from 1 swagger version to another.  We can debate the naming of this later.  Lets keep this change restricted only only the _required_ changes to include the new swagger version.  The unnecessary churn makes reviewing harder.